msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/TestCase832450.java

```diff
@@ -0,0 +1,162 @@
+                mApplication,
+                parameters,
+                new OnInteractionRequired() {
+                    @Override
+                    public void handleUserInteraction() {
+                        final PromptHandlerParameters promptHandlerParameters = PromptHandlerParameters.builder()
+                                .prompt(PromptParameter.SELECT_ACCOUNT)
+                                .loginHint(mLoginHint)
+                                .sessionExpected(false)
+                                .consentPageExpected(false)
+                                .speedBumpExpected(false)
+                                .broker(getBroker())
+                                .expectingBrokerAccountChooserActivity(true)
+                                .build();
+
+                        new AadPromptHandler(promptHandlerParameters)
+                                .handlePrompt(username, password);
+                    }
+                }
+        );
+
+        interactiveRequest.execute();
+        latch.await();
+
+        final IAccount account = getAccount();
+
+        // acquiring token silently
+        final CountDownLatch silentLatch = new CountDownLatch(1);
+
+        final AcquireTokenSilentParameters silentParameters = new AcquireTokenSilentParameters.Builder()
+                .forAccount(account)
+                .fromAuthority(account.getAuthority())
+                .forceRefresh(false)
+                .withScopes(Arrays.asList(mScopes))
+                .withCallback(successfulSilentCallback(silentLatch))
+                .build();
+
+        mApplication.acquireTokenSilentAsync(silentParameters);
+        silentLatch.await();
+
+        // forwarding time for 1 day.
+        TestContext.getTestContext().getTestDevice().getSettings().forwardDeviceTimeForOneDay();
+
+        // acquiring token silently
+        final CountDownLatch refreshTokenLatch = new CountDownLatch(1);
+
+        final AcquireTokenSilentParameters refreshSilentParameters = new AcquireTokenSilentParameters.Builder()
+                .forAccount(account)
+                .fromAuthority(account.getAuthority())
+                .forceRefresh(true)
+                .withScopes(Arrays.asList(mScopes))
+                .withCallback(successfulSilentCallback(refreshTokenLatch))
+                .build();
+
+        mApplication.acquireTokenSilentAsync(refreshSilentParameters);
+        refreshTokenLatch.await();
+    }
+
+    @Override
+    public String[] getScopes() {
+        return new String[]{"user.read"};
+    }
+
+    @Override
+    public String getAuthority() {
+        return mApplication.getConfiguration().getDefaultAuthority().toString();
+    }
+
+    @Override
+    public int getConfigFileResourceId() {
+        return R.raw.msal_config_default;
+    }
+
+    @NonNull
+    @Override
+    public ITestBroker getBroker() {
+        return new BrokerMicrosoftAuthenticator();
+    }
```

Comment:
You no longer need to override this

Reply:
removed getBroker() function from testcase